### DISCUSSION TITLE
Trying to implement the s735.json to Nibepi

### DISCUSSION THREAD

**Discussion by Denna93** [yozik04/nibe#123] [Q&A]
Hey. I am trying to implement your s735.json to the project NibePi. The Json for the different models needs the mode"R" value. If its Read only or R/W read-write. Do you have this documentation?

**Comment by yozik04** [comment]
I think we use register export from the unit for S-series. Am I right, @elupus?

**Comment by elupus** [comment]
Yup. Set pump to english, plug in a usb memory, and export the whole parameter set.
